### PR TITLE
feat: require newline at the end of files

### PR DESCRIPTION
**Pull request by janecakemaster**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -77,5 +77,6 @@ module.exports = {
     // Require consistent return values (either always or never specifying values) so we don't
     // rely on implicit returns when return booleans or undefined.
     'consistent-return': ['error', { treatUndefinedAsUnspecified: true }],
+    'eol-lst': ['error', 'always'],
```
Comment (skeggse) [inline index.js]:
```suggestion
    'eol-last': ['error', 'always'],
```

Comment (janecakemaster) [reply index.js]: OH i knew that looked weird. thanks

**Comment by janecakemaster** [comment]
@wearhere this will handle non-sublime users. it just got me bc i forgot to set up in vscode, but it would be nice if we just never had to remember to do it :)

**Comment by skeggse** [comment]
ideally you'd still configure your editor to add the trailing newline and this would just be to help you remember, right?

**Pull request merged**